README.md

```diff
@@ -111,11 +111,11 @@ environment variables that are required. Most importantly, they call the
 To build, run the following:
 
 ```
-cd $SOURCE_DIR
+cd %SOURCE_DIR%
 mkdir build
 cd build
 ..\env_win64.bat
-cmake .. -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR -DCMAKE_BUILD_TYPE=Release \
```

Comment:
The multiline command is written with `^` instead of `\` in Batch.

Reply:
Good to know. I made it one line in my build scripts.

Reply:
DONE